### PR TITLE
Add package structure

### PR DESCRIPTION
**Pull request by jdhoffa**
This first commit adds typical R package boilerplate, and an explanatory README.

### CODE REVIEW

**Comment by jdhoffa** [comment]
Note: All R CMD checks will fail, because there aren't actually any functions or tests yet.

**Comment by cjyetman** [comment]
Is this a replacement for pacta.scenario.preparation?

**Comment by jdhoffa** [comment]
> Is this a replacement for pacta.scenario.preparation?

That is the vision yeah, but it's entirely boilerplate now (s

Main difference is this one is public :-)

(I figured just make a new one with a new name, rather than go through the weird crap of deprecating the old private one). 

Once it is a sufficient replacement, we can archive the old one

File: .gitignore [hunk 1]
```diff
@@ -47,3 +47,4 @@ po/*~
 
 # RStudio Connect folder
 rsconnect/
+.Rproj.user
```
Comment (cjyetman) [inline .gitignore:50]:
```suggestion
.Rproj.user
.DS_Store
```
🙏🏻

Comment (cjyetman) [reply .gitignore:50]:
```suggestion
.Rproj.user

# Annoying OS X artifacts
.DS_Store

```
or better

Comment (cjyetman) [reply .gitignore:50]:
```suggestion
.Rproj.user

# Annoying macOS artifacts
.DS_Store

```
or even better

File: README.Rmd [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
```
Comment (cjyetman) [inline README.Rmd:5]: This should not be here?

**Review by cjyetman — APPROVED** [review]

File: README.Rmd (hunk 2 above)
Comment (cjyetman) [reply README.Rmd:5]: Oh, I was confused by the "please edit that file" which is referring to THIS file

**Review by cjyetman — APPROVED** [review]

**Pull request merged**